[Solo v0.73.0] Diagnostic Report - solo-deployment - 2026-05-19T12-24-34

## Solo Diagnostic Report

- **Solo Version**: 0.73.0
- **Deployment**: solo-deployment
- **Timestamp**: 2026-05-19T12-24-34
- **Platform**: darwin 25.5.0
- **Node.js**: v26.0.0
- **Diagnostic logs**: /Users/ibankov/.solo/logs/hiero-components-logs
- **Debug archive**: /Users/ibankov/.solo/solo-debug-solo-deployment-2026-05-19T12-24-34.zip

## Diagnostics Analysis

```
Solo Diagnostics Analysis Report
Generated: 2026-05-19T12:24:34.278Z

Detected 8 potential issue(s):

1. [Exception Stack] Exception detected in swirlds.log
   Source: network-node1-0-log-config.zip:output/swirlds.log
   Evidence:
   - 2026-05-19 12:12:22.759 3        ERROR EXCEPTION        <main> Log4jSetup: exception on thread main
   - java.io.UncheckedIOException: Property file semantic-version.properties could not be found
   - 	at com.hedera.node.config.sources.PropertyConfigSource.loadProperties(PropertyConfigSource.java:53)
   - 	at com.hedera.node.config.sources.PropertyConfigSource.<init>(PropertyConfigSource.java:40)
   - 	at com.hedera.node.app.config.BootstrapConfigProviderImpl.<init>(BootstrapConfigProviderImpl.java:65)
   - 	at com.hedera.node.app.config.BootstrapConfigProviderImpl.<init>(BootstrapConfigProviderImpl.java:51)
   - 	at com.hedera.node.app.Hedera.<init>(Hedera.java:525)
   - 	at com.hedera.node.app.ServicesMain.newHedera(ServicesMain.java:336)
   - 	at com.hedera.node.app.ServicesMain.main(ServicesMain.java:169)
   - Caused by: java.io.IOException
   - 	... 7 more

2. [Exception Stack] Exception detected in swirlds.log
   Source: network-node2-0-log-config.zip:output/swirlds.log
   Evidence:
   - 	Caused by: java.net.SocketException: Connection or outbound has closed
   - 		at java.base/sun.security.ssl.SSLSocketImpl$AppOutputStream.write(SSLSocketImpl.java:1299)
   - 		at org.hiero.consensus.io.counting.CountingOutputStream.write(CountingOutputStream.java:58)
   - 		at java.base/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:123)
   - 		at java.base/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:203)
   - 		at java.base/java.io.DataOutputStream.flush(DataOutputStream.java:142)
   - 		at org.hiero.consensus.gossip.impl.network.protocol.rpc.RpcPeerProtocol.writeMessages(RpcPeerProtocol.java:400)
   - 		at org.hiero.consensus.gossip.impl.network.protocol.rpc.RpcPeerProtocol.lambda$runProtocol$1(RpcPeerProtocol.java:307)
   - 		at org.hiero.base.concurrent.ThrowingRunnable.call(ThrowingRunnable.java:24)
   - 		at org.hiero.base.concurrent.ThrowingRunnable.call(ThrowingRunnable.java:9)
   - 		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:328)
   - 		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1090)
   - 		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:614)
   - 		... 2 more

3. [Exception Stack] Exception detected in swirlds.log
   Source: network-node3-0-log-config.zip:output/swirlds.log
   Evidence:
   - 	Caused by: java.net.SocketException: Connection or outbound has closed
   - 		at java.base/sun.security.ssl.SSLSocketImpl$AppOutputStream.write(SSLSocketImpl.java:1299)
   - 		at org.hiero.consensus.io.counting.CountingOutputStream.write(CountingOutputStream.java:58)
   - 		at java.base/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:123)
   - 		at java.base/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:203)
   - 		at java.base/java.io.DataOutputStream.flush(DataOutputStream.java:142)
   - 		at org.hiero.consensus.gossip.impl.network.protocol.rpc.RpcPeerProtocol.writeMessages(RpcPeerProtocol.java:400)
   - 		at org.hiero.consensus.gossip.impl.network.protocol.rpc.RpcPeerProtocol.lambda$runProtocol$1(RpcPeerProtocol.java:307)
   - 		at org.hiero.base.concurrent.ThrowingRunnable.call(ThrowingRunnable.java:24)
   - 		at org.hiero.base.concurrent.ThrowingRunnable.call(ThrowingRunnable.java:9)
   - 		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:328)
   - 		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1090)
   - 		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:614)
   - 		... 2 more

4. [Exception Stack] Exception detected in swirlds.log
   Source: network-node4-0-log-config.zip:output/swirlds.log
   Evidence:
   - 	Caused by: java.net.SocketException: Connection or outbound has closed
   - 		at java.base/sun.security.ssl.SSLSocketImpl$AppOutputStream.write(SSLSocketImpl.java:1299)
   - 		at org.hiero.consensus.io.counting.CountingOutputStream.write(CountingOutputStream.java:58)
   - 		at java.base/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:123)
   - 		at java.base/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:203)
   - 		at java.base/java.io.DataOutputStream.flush(DataOutputStream.java:142)
   - 		at org.hiero.consensus.gossip.impl.network.protocol.rpc.RpcPeerProtocol.writeMessages(RpcPeerProtocol.java:400)
   - 		at org.hiero.consensus.gossip.impl.network.protocol.rpc.RpcPeerProtocol.lambda$runProtocol$1(RpcPeerProtocol.java:307)
   - 		at org.hiero.base.concurrent.ThrowingRunnable.call(ThrowingRunnable.java:24)
   - 		at org.hiero.base.concurrent.ThrowingRunnable.call(ThrowingRunnable.java:9)
   - 		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:328)
   - 		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1090)
   - 		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:614)
   - 		... 2 more

5. [Application Error] Application ERROR detected in pod log: mirror-1-importer-6796dcdcff-xb2pv
   Source: kind-solo/mirror-1-importer-6796dcdcff-xb2pv.log
   Evidence:
   - line 220: 2026-05-19T11:59:45.454Z  INFO main o.f.c.i.c.DbMigrate Migrating schema "public" to version "1.78.0 - recoverable error nullables"

6. [Application Error] Application ERROR detected in pod log: mirror-1-rest-79f74ccdf4-vdzhp
   Source: kind-solo/mirror-1-rest-79f74ccdf4-vdzhp.log
   Evidence:
   - line 13: 2026-05-19T11:59:34.581Z ERROR Startup Error connecting to redis://***:***@solo-shared-resources-redis:6379: connect ECONNREFUSED 10.96.129.80:6379
   - line 14: 2026-05-19T11:59:35.771Z ERROR Startup healthcheck failed
   - line 15: Error: password authentication failed for user "mirror_rest"
   - line 19: 2026-05-19T11:59:36.583Z ERROR Startup Error connecting to redis://***:***@solo-shared-resources-redis:6379: connect ECONNREFUSED 10.96.129.80:6379
   - line 20: 2026-05-19T11:59:36.768Z ERROR Startup healthcheck failed
   - line 21: Error: password authentication failed for user "mirror_rest"
   - line 25: 2026-05-19T11:59:36.773Z ERROR Startup healthcheck failed
   - line 26: Error: password authentication failed for user "mirror_rest"

7. [Application Error] Application ERROR detected in pod log: solo-shared-resources-postgres-0
   Source: kind-solo/solo-shared-resources-postgres-0.log
   Evidence:
   - line 30: 2026-05-19 11:59:35.771 GMT [246] FATAL:  password authentication failed for user "mirror_rest"
   - line 33: 2026-05-19 11:59:36.768 GMT [247] FATAL:  password authentication failed for user "mirror_rest"
   - line 36: 2026-05-19 11:59:36.773 GMT [248] FATAL:  password authentication failed for user "mirror_rest"
   - line 39: 2026-05-19 11:59:37.774 GMT [249] FATAL:  password authentication failed for user "mirror_rest"
   - line 42: 2026-05-19 11:59:38.775 GMT [264] FATAL:  password authentication failed for user "mirror_rest"
   - line 45: 2026-05-19 11:59:39.773 GMT [265] FATAL:  password authentication failed for user "mirror_rest"
   - line 48: 2026-05-19 11:59:40.774 GMT [266] FATAL:  password authentication failed for user "mirror_rest"
   - line 51: 2026-05-19 11:59:41.774 GMT [267] FATAL:  password authentication failed for user "mirror_rest"

8. [Application Error] ERROR detected in solo.log
   Source: ../solo.log
   Evidence:
   - line 8632: [15:00:12.844] ERROR: Error in upgrade network: node 'node1' is not ACTIVE[ attempt = 300/300 ]
   - line 8633: err: {
   - line 8634: "type": "SoloError",
   - line 8635: "message": "Error in upgrade network: node 'node1' is not ACTIVE[ attempt = \u001b[94m300/300\u001b[39m ]: node 'node1' is not ACTIVE[ attempt = \u001b[94m300/300\u001b[39m ]",
   - line 8636: "stack":
   - line 8637: SoloError: Error in upgrade network: node 'node1' is not ACTIVE[ attempt = 300/300 ]
   - line 8638: at NodeCommandHandlers.commandAction (file:///opt/homebrew/Cellar/solo/0.73.0/libexec/lib/node_modules/@hashgraph/solo/src/core/command-handler.ts:59:15)
   - line 8639: at runNextTicks (node:internal/process/task_queues:65:5)
   - line 8640: at listOnTimeout (node:internal/timers:567:9)
   - line 8641: at processTimers (node:internal/timers:541:7)
   - line 8642: at NodeCommandHandlers.upgrade (file:///opt/homebrew/Cellar/solo/0.73.0/libexec/lib/node_modules/@hashgraph/solo/src/commands/node/handlers.ts:490:5)
   - line 8643: at Object.handler (file:///opt/homebrew/Cellar/solo/0.73.0/libexec/lib/node_modules/@hashgraph/solo/src/core/command-path-builders/command-builder.ts:119:47)
   - line 8644: caused by: SoloError: node 'node1' is not ACTIVE[ attempt = 300/300 ]
   - line 8645: at NodeCommandTasks.checkNetworkNodeActiveness (file:///opt/homebrew/Cellar/solo/0.73.0/libexec/lib/node_modules/@hashgraph/solo/src/commands/node/tasks.ts:658:13)

```

## Description

_Please describe the issue you encountered..._

## Steps to Reproduce

_Please list the steps to reproduce the issue..._

---
_Note: A debug archive was generated at `/Users/ibankov/.solo/solo-debug-solo-deployment-2026-05-19T12-24-34.zip`. Please attach it to this issue via the GitHub web interface._

--- a/src/commands/node/handlers.ts
+++ b/src/commands/node/handlers.ts
@@ -282,6 +282,7 @@ export class NodeCommandHandlers extends CommandHandler {
   private upgradeExecuteTasks(): SoloListrTask<NodeUpgradeContext>[] {
     return [
       this.tasks.checkAllNodesAreFrozen('existingNodeAliases'),
+      this.tasks.stopNodes('existingNodeAliases'),
       this.tasks.downloadNodeUpgradeFiles(),
       this.tasks.getNodeLogsAndConfigs(),
       this.tasks.upgradeNodeConfigurationFilesWithChart(),

--- a/src/commands/node/tasks.ts
+++ b/src/commands/node/tasks.ts
@@ -2246,7 +2246,7 @@ export class NodeCommandTasks {
 
   public stopNodes(
     nodeAliasesProperty: string,
-  ): SoloListrTask<NodeStopContext | NodeFreezeContext | NodeDestroyContext> {
+  ): SoloListrTask<NodeStopContext | NodeFreezeContext | NodeDestroyContext | NodeUpgradeContext> {
     return {
       title: 'Stopping nodes',
       task: async (context_, task): Promise<any> => {

